testing/app.go

```diff
@@ -27,7 +27,7 @@ import (
 	"github.com/cosmos/ibc-go/v3/testing/simapp"
 )
 
-var DefaultTestingAppInit func() (TestingApp, map[string]json.RawMessage) = SetupTestingApp
+var DefaultTestingAppInit = SetupTestingApp
```

Comment:
why was the type declaration deleted here?

Reply:
Revive let me know that it was not needed, I think because the type is implied on the other side of =